src/assets/icons/chevron.svg

```diff
@@ -1,3 +1 @@
-<svg width="14px" height="14px" viewBox="0 0 14 14" version="1.1" xmlns="http://www.w3.org/2000/svg">
-  <path d="M7.97487373,9.8890873 L3.73223305,14.131728 C3.34170876,14.5222523 2.70854378,14.5222523 2.31801948,14.131728 C1.92749519,13.7412037 1.92749519,13.1080387 2.31801948,12.7175144 L7.26776695,7.76776695 C7.65829124,7.37724266 8.29145622,7.37724266 8.68198052,7.76776695 L13.631728,12.7175144 C14.0222523,13.1080387 14.0222523,13.7412037 13.631728,14.131728 C13.2412037,14.5222523 12.6080387,14.5222523 12.2175144,14.131728 L7.97487373,9.8890873 Z M7.97487373,3.8890873 L3.73223305,8.13172798 C3.34170876,8.52225228 2.70854378,8.52225228 2.31801948,8.13172798 C1.92749519,7.74120369 1.92749519,7.10803871 2.31801948,6.71751442 L7.26776695,1.76776695 C7.65829124,1.37724266 8.29145622,1.37724266 8.68198052,1.76776695 L13.631728,6.71751442 C14.0222523,7.10803871 14.0222523,7.74120369 13.631728,8.13172798 C13.2412037,8.52225228 12.6080387,8.52225228 12.2175144,8.13172798 L7.97487373,3.8890873 Z" transform="translate(7.974874, 7.949747) rotate(-90.000000) translate(-7.974874, -7.949747) "></path>
-</svg>
\ No newline at end of file
+<svg width="14" height="14" xmlns="http://www.w3.org/2000/svg"><path d="M9.914 7.95l4.243 4.242a1 1 0 1 1-1.414 1.415l-4.95-4.95a1 1 0 0 1 0-1.414l4.95-4.95a1 1 0 0 1 1.414 1.414L9.914 7.95zm-6 0l4.243 4.242a1 1 0 1 1-1.414 1.415l-4.95-4.95a1 1 0 0 1 0-1.414l4.95-4.95a1 1 0 1 1 1.414 1.414L3.914 7.95z"/></svg>
```

Comment:
It is unclear if we are using this `svg`s for the icons on the legend or the `vizzuality-components` ones.
Maybe we can get rid of this until we are able to use them (to avoid forgetting it if we do not finally get to config svg sprite loader on webpack config )